exporter/elasticsearchexporter/data_stream_router.go

```diff
@@ -164,6 +164,13 @@ func routeRecord(
 	// 4. use default hardcoded data_stream.*
 	dataset, datasetExists := getFromAttributes(elasticsearch.DataStreamDataset, defaultDataStreamDataset, recordAttr, scopeAttr, resourceAttr)
 	namespace, namespaceExists := getFromAttributes(elasticsearch.DataStreamNamespace, defaultDataStreamNamespace, recordAttr, scopeAttr, resourceAttr)
+
+	datastreamType := defaultDSType
```

Comment:
nit: let's use either `dsType` or `dataStreamType` to keep naming consistent with the rest of the code.